grails-app/jobs/NodeActivatorJob.groovy

```diff
@@ -72,49 +71,58 @@ class NodeActivatorJob {
         try{
             if (log.debugEnabled) {
                 sw?.start('Finding active and running FlowStatuses')

```

Comment:
If you don't have code between lines 73 and 74 which executes some heavy task, then it doesn't make sense to start and stop the stop watch.